xml/System/Memory`1.xml

```diff
@@ -22,9 +22,34 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of items in the <see cref="T:System.Memory`1" />. </typeparam>
+    <summary>Represents a contiguous region of memory. </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+  
+Like <xref:System.Span%601>, `Memory<T>` represents a contiguous region of memory. Unlike <xref:System.Span%601>, however, `Memory<T>` is not a [ref struct](~/doocs/csharp/reference-semantics-with-value-types?view=netcore-2.1.md#ref-struct-type). This means that `Memory<T>` can be backed by:
+
+- An array.
+- A string (a sequence of characters).
+- A <xref:System.Runtime.InteropServices.SafeHandle>.
+
+`Memory<T>` cannot be backed by transient unmanaged memory, such as memory available by using [stackalloc](~/docs/csharp/language-reference/keywords/stackalloc.md).
```

Comment:
This isn't really true.  With `IMemoryPool<T>`, a `Memory<T>` can be created that's backed pretty much any contiguous memory, regardless of where it came from: if you can get a pointer to it, you can create a `Memory<T>` for it... it just requires some more work to do.